### PR TITLE
Added Czech translations

### PR DESCRIPTION
**Pull request by Daybringer**
## What does this PR do?

Providing Czech translation of email templates and country names.

## Test Plan

I have tested the translation through GET requests.

## Related PRs and Issues

(https://github.com/appwrite/appwrite/issues/1679)[]

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
yes

### CODE REVIEW

**Comment by christyjacob4** [comment]
Can we fix the issue link in your PR description ?

**Comment by Daybringer** [comment]
Yeah, of course.

File: app/config/locale/translations/cs.json [hunk 1]
```diff
@@ -1,232 +1,232 @@
...
-    "countries.sc": "Seychely",
-    "countries.sy": "Sýrie",
-    "countries.td": "Čad",
-    "countries.tg": "Togo",
-    "countries.th": "Thajsko",
-    "countries.tj": "Tádžikistán",
-    "countries.tm": "Turkmenistán",
-    "countries.tl": "Timor-Leste",
-    "countries.to": "Tonga",
-    "countries.tt": "Trinidad a Tobago",
```
Comment (christyjacob4) [inline app/config/locale/translations/cs.json:205]: The translations itself look good but it seems like we have introduced a lot of other formatting in the process. Can we fix these space/ indentation changes to ensure consistency with other language templates?

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

File: app/config/locale/translations/cs.json (hunk 1 above)
Comment (Daybringer) [reply app/config/locale/translations/cs.json:205]: Yeah, of course.

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Screenshot from 2021-10-06 16-07-24

Im talking about these changes. Can we fix these ?

**Comment by christyjacob4** [comment]
Hey @Daybringer  Let us know if you're still working on these. Else we'll need to close this 👍

**Pull request closed without merging**